requirements-dev.txt

```diff
@@ -27,4 +31,5 @@ BeautifulSoup4
 
 requests
 
-lxml
\ No newline at end of file
+lxml
```

Comment:
Is it still required?

Reply:
We need it because pytest always runs all tests for all homeworks.